### ISSUE TITLE
Getting file not found error during xctest cases build

### ISSUE THREAD

**Issue by Girishav** [CocoaLumberjack/CocoaLumberjack#914]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

 am facing compile time issue during unit test cases build. getting following error in one of my class file.

![screen shot 2018-02-28 at 10 50 04 am](https://user-images.githubusercontent.com/14818028/36779399-de54631e-1c95-11e8-83de-ce30de8238fe.png)

we are using xmppframework in project as pod dependency which inturn uses cocoa lumberjack . didnt get any issue while building other targets. problem is only with xctest cases only…

here is my pod file, adding `xmppframework` to both targets, but still issue persists…
```ruby
target ‘MySdk' do
  use_frameworks!
  pod 'XMPPFramework', '~> 3.7.0'
  target 'MySdkTests' do
    inherit! :search_paths 
  end
end
```
any help would be appreciated. i have been struck with this issue for past few days…

**Comment by bpoplauschi** [comment]
@Girishav thanks for your interest in our project.
I've re-added the template issue info to the description, you need to fill that info so we can better understand how you are using the library.

One idea is to move `use_frameworks!` outside of the `MySdk` target.

**Comment by sushichop** [comment]
@Girishav 
We still need detailed info based on template issue.
But I created a small sample project(Cocoa Touch framework with Objective-C) and it worked fine
(I could import `<CocoaLumberjack/CocoaLumberjack.h>` and build project even at unit tests target).

Podfile is here.

```ruby
use_frameworks!

target 'LumberjackXMPPSample01' do
  
  pod 'XMPPFramework', '4.0.0'

  target 'LumberjackXMPPSample01Tests' do
    inherit! :search_paths
    # Pods for testing
  end

end
``` 

Can you try this?

**Comment by sushichop** [comment]
Oops! Sorry, I may have misled everyone.
 I forgot to paste Podfile in the case of `XMPPFramework, ~> 3.7.0`😅

Actually I could built and import `<CocoaLumberjack/CocoaLumberjack.h>` in both cases  `'XMPPFramework', '4.0.0'` and '`XMPPFramework', '~> 3.7.0'`.

Podfile is here('`XMPPFramework', '~> 3.7.0'`).

```
platform :ios, '10.0'
use_frameworks!

target 'LumberjackXMPPSample' do
  
  pod 'XMPPFramework', '~> 3.7.0'

  target 'LumberjackXMPPSampleTests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```

**Comment by sushichop** [comment]
@bpoplauschi 
So I suspect one of the causes was `CocoaLumberjack` version, probably.
Any Idea?

**Comment by bpoplauschi** [comment]
@sushichop might be that, or another caused that `XMPPFramework` have fixed. As long as it works ... :)
